domains/POAS/events/lpa-updated/examples/example.json

```diff
@@ -0,0 +1,4 @@
+{
+  "uid": "M-1234-5678-9012",
```

Comment:
Should this include a createdAt field?

Reply:
That's a good point, I'll raise it in the dev channel although maybe the timestamp would be enough in the metadata